README.md

```diff
@@ -131,9 +131,124 @@ You can add your customized Python code to process your input and output data. T
 1. The customized Python code file should be named `inference.py` and it should be under `code` directory of your model archive.
 2. `inference.py` should implement either a pair of `input_handler` and `output_handler` functions or a single `handler` function. Note that if `handler` function is implemented, `input_handler` and `output_handler` will be ignored.
 
+To implement pre/post-processing handler(s), you will need to make use of the `Context` object created by Python service. The `Context` is a `namedtuple` with following attributes:
+- `model_name`: the name of the model you will to use for inference, for example 'half_plus_three'
+- `model_version`: version of the model
+- `method`: inference method, for example, 'predict', 'classify' or 'regress'
```

Comment:
Please add link to TFS documentation https://www.tensorflow.org/tfx/serving/api_rest#classify_and_regress_api

Reply:
done